neptune/client.py

```diff
@@ -703,9 +787,14 @@ def _upload_loop_chunk(self, fun, part, data, **kwargs):
                    },
                    **kwargs)
 
-    def _upload_raw_data(self, experiment, api_method, data, headers):
-        url = self.api_address + api_method.operation.path_name
-        url = url.replace("{experimentId}", experiment.internal_id)
+    def _upload_raw_data(self, api_method, data, headers, path_params, query_params):
```

Comment:
Shouldn't content-type header be set here?

Reply:
Caller of this method is responsible for setting all necessary headers.